crates/starknet/src/starknet/mod.rs

```diff
@@ -159,6 +175,95 @@ impl Starknet {
+                            .get(&tx.class_hash)
+                            .expect("Failed to load Cairo0ContractClass from state");
+                        if let ContractClass::Cairo0(contract) = contract_class {
+                            let declare_tx = BroadcastedDeclareTransactionV1::new(
+                                tx.sender_address,
+                                tx.max_fee,
+                                &tx.signature,
+                                tx.nonce,
+                                contract,
+                                tx.version,
+                            );
+                            let result = this.add_declare_transaction_v1(declare_tx);
+                            if result.is_err() {
+                                panic!("Failed to add BroadcastedDeclareTransactionV1");
+                            }
+                        } else {
+                            panic!("Failed to load Cairo0ContractClass");
+                        };
+                    }
+                    Transaction::Declare(DeclareTransaction::Version2(tx)) => {
+                        let declare_tx = BroadcastedDeclareTransactionV2::new(
+                            &tx.contract_class,
+                            tx.compiled_class_hash,
+                            tx.sender_address,
+                            tx.max_fee,
+                            &tx.signature,
+                            tx.nonce,
+                            tx.version,
+                        );
+                        let result = this.add_declare_transaction_v2(declare_tx);
+                        if result.is_err() {
+                            panic!("Failed to add BroadcastedDeclareTransactionV2");
+                        }
+                    }
+                    Transaction::DeployAccount(tx) => {
+                        let deploy_account_tx = BroadcastedDeployAccountTransaction::new(
+                            &tx.constructor_calldata,
+                            tx.max_fee,
+                            &tx.signature,
+                            tx.nonce,
+                            tx.class_hash,
+                            tx.contract_address_salt,
+                            tx.version,
+                        );
+                        println!("DeployAccount!!!!");
+                        println!("DeployAccount!!!!");
+                        println!("DeployAccount!!!!");
+
+                        let result = this.add_deploy_account_transaction(deploy_account_tx);
+                        if result.is_err() {
+                            panic!("Failed to add BroadcastedDeployAccountTransaction");
+                        }
+                    }
+                    Transaction::Deploy(_tx) => {
+                        panic!("DeployTransaction is not supported");
+                    }
+                    Transaction::Invoke(InvokeTransaction::Version0(_tx)) => {
```

Comment:
```suggestion
                    Transaction::Invoke(InvokeTransaction::Version0(_)) => {
```

Reply:
fixed in aca53690763aa8ccba3a03b49830d8cc260ae093